### PR TITLE
Expose ISTIO_META_MESH_ID and ISTIO_META_CLUSTER_ID in helm and glooctl

### PR DESCRIPTION
**Pull request by kevin-shelaga**
# Description

Expose ISTIO_META_MESH_ID and ISTIO_META_CLUSTER_ID in helm and glooctl

# Context

PR for https://github.com/solo-io/gloo/issues/4325

# Checklist:

- [x] I included a concise, user-facing changelog (for details, see https://github.com/solo-io/go-utils/tree/master/changelogutils) which references the issue that is resolved.
- [x] If I updated APIs (our protos) or helm values, I ran `make install-go-tools generated-code` to ensure there will be no code diff
- [x] I followed guidelines laid out in the Gloo Edge [contribution guide](https://docs.solo.io/gloo-edge/latest/contributing/)
- [x] I opened a draft PR or added the work in progress label if my PR is not ready for review
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
BOT NOTES: 
resolves https://github.com/solo-io/gloo/issues/4325

### CODE REVIEW

**Comment by Sodman** [comment]
/test
Thanks for taking this on!

File: projects/gloo/cli/pkg/cmd/options/options.go [hunk 1]
```diff
@@ -159,6 +159,8 @@ type Add struct {
 type Istio struct {
 	Upstream  string // upstream for which we are changing the istio mTLS settings
 	Namespace string // namespace in which istio is installed
+	IstioMetaMeshId string // IstioMetaMeshId sets ISTIO_META_MESH_ID env var
+	IstioMetaClusterId string // IstioMetaMeshId sets ISTIO_META_CLUSTER_ID env var
```
Comment (Sodman) [inline projects/gloo/cli/pkg/cmd/options/options.go:163]: Minor copypaste typo
```suggestion
	IstioMetaClusterId string // IstioMetaClusterId sets ISTIO_META_CLUSTER_ID env var
```

Comment (kevin-shelaga) [reply projects/gloo/cli/pkg/cmd/options/options.go]: Oops. Good catch, thank you :)

**Comment by Sodman** [comment]
Looks good! I believe you'll just need to re-run codegen to make `go fmt` happy with the new fields in the Istio cli options, then this should pass the codegen check.

**Comment by kevin-shelaga** [comment]
@Sodman I fixed the helm docs too, I noticed a typo there. Saved all and regenerated docs. Hopefully good to go now.

**Review by Sodman — APPROVED** [review]

**Comment by Sodman** [comment]
Thanks again for the contribution @kevin-shelaga!

**Pull request merged**